react/features/filmstrip/subscriber.web.js

```diff
@@ -19,6 +19,7 @@ import {
     setTileViewDimensions,
     setVerticalViewDimensions
 } from './actions';
+import { setScreenshareFilmstripParticipant } from './actions.web';
```

Comment:
Maybe you can add this import to the one above?

Reply:
Normally I would agree, but when we convert a file to TypeScript we're gonna need all imports to be specific (function.web instead of functions etc). I think by leaving them like this it could help when these files are converted to TS

Reply:
Ok. But then let's convert the top one for consistency. I don't mind having only actions.web. Having actions and actions.web just seems weird.